VideoLocker/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java

```diff
@@ -211,21 +213,25 @@ public ProfileModel call( ) {
 
                 // do SOCIAL LOGIN first
                 AuthResponse social = null;
+                HashMap<String, CharSequence> descParams = new HashMap<>();
+                String platformName = environment.getConfig().getPlatformName();
+                descParams.put("platform_name", environment.getConfig().getPlatformName());
+                descParams.put("platform_destination", environment.getConfig().getPlatformDestinationName());
```

Comment:
We don't have too many strings with both these parameters and it's sort of specific to the individual translation so I feel kind of weird about it.